manifests/pipeline.yaml

```diff
@@ -285,3 +285,27 @@ objects:
         jenkinsPipelineStrategy:
           type: JenkinsPipeline
           jenkinsfilePath: Jenkinsfile.release
+
+  ### FEDORA COREOS STREAM PIPELINE ###
+
+  - kind: BuildConfig
+    apiVersion: "build.openshift.io/v1"
+    metadata:
+      name: fedora-coreos-pipeline-stream
+      annotations:
+        coreos.com/deploy-default: "false"
+        coreos.com/s3-bucket: ${S3_BUCKET}
```

Comment:
This doesn't seem to be used

Reply:
Good catch, fixed it! was using it initially to test aws upload but forgot during clean-up.